src/_pytest/skipping.py

```diff
@@ -204,14 +204,46 @@ def pytest_terminal_summary(terminalreporter):
             tr._tw.line(line)
 
 
+def _get_line_with_reprcrash_message(config, rep, termwidth):
+    """Get summary line for a report, trying to add reprcrash message."""
+    verbose_word = _get_report_str(config, rep)
+    pos = _get_pos(config, rep)
+
+    line = "%s %s" % (verbose_word, pos)
+    len_line = len(line)
+    ellipsis, len_ellipsis = "...", 3
+    if len_line > termwidth - len_ellipsis:
+        # No space for an additional message.
+        return line
+
+    try:
+        msg = rep.longrepr.reprcrash.message
+    except AttributeError:
+        pass
+    else:
+        # Only use the first line.
+        i = msg.find("\n")
+        if i != -1:
+            msg = msg[:i]
+        len_msg = len(msg)
+
+        sep, len_sep = " - ", 3
+        max_len_msg = termwidth - len_line - len_sep
+        if max_len_msg >= len_ellipsis:
+            if len_msg > max_len_msg:
+                msg = msg[: (max_len_msg - len_ellipsis)] + ellipsis
+            line += sep + msg
```

Comment:
Oh just realized this: is `msg` unicode-safe in Python 2? because we will be joining a `str/bytes` with a potential unicode message, so there's a good chance this might blow up

Reply:
Would appreciate a failing example..  do you mean we should prefix `sep` etc also with `u`?

Reply:
Hmm actually no, this is safe: if `msg` is unicode, Python 2 will try to elevate `sep` to unicode using ascii (which is fine), not the other way around. Sorry for the brain fart. 👍

Reply:
If you want to play it safe and ensure we don't regress though, you can change the message to use a smiling face emoji instead (😄), but is up to you. 👍

Reply:
Added.. but it shows that we're not handling terminal cells there really - i.e. it would wrap due to this.

Reply:
Oh. Hmm not even sure what the solution would be, TBH.

Reply:
added a commit using wcwidth.. not sure if it is worth the extra dependency, but it worked well.. ;)

Reply:
I think adding `wcwidth` is fine: small, license is OK, seems to be maintained